src/components/pipelineRuns-list/PipelineRunsList.tsx

```diff
@@ -44,12 +45,35 @@ const PipelineRunsList: React.FC<PipelineRunsListProps> = ({
     ? 5
     : 4;
 
-  const [pipelineRuns, pipelineRunsLoaded, pipelineRunsLoadError] =
-    useGetPipelineRuns(namespace, { name: PLRsForName, kind: PLRsForKind });
+  const [
+    pipelineRuns,
+    pipelineRunsLoaded,
+    pipelineRunsLoadError,
+    nextPageToken,
+  ] = useGetPipelineRuns(namespace, { name: PLRsForName, kind: PLRsForKind });
   const [data, filteredData, onFilterChange] = useListPageFilter(
     pipelineRuns,
     filters,
   );
+
+  // Once OCPBUGS-43538 ticket is closed, use onRowsRendered prop in VirtualizedTable for load more on scroll
+  React.useEffect(() => {
```

Comment:
Can we create a custom hook for it and use it in both the PLR and TR list pages?